x-pack/plugins/security/public/account_management/user_profile/user_profile_api_client.ts

```diff
@@ -10,9 +10,34 @@ import { Subject } from 'rxjs';
 
 import type { HttpStart } from '@kbn/core/public';
 
-import type { GetUserProfileResponse, UserProfileData } from '../../../common';
+import type { GetUserProfileResponse, UserProfile, UserProfileData } from '../../../common';
 
-const USER_PROFILE_URL = '/internal/security/user_profile';
```

Comment:
__note:__ we try not to use constants for the HTTP paths (or construct paths dynamically) as it makes searching for the "usages" of the particular APIs incredibly painful.